README.md

```diff
@@ -1,9 +1,9 @@
 # Azure Plugins for Gradle 
 This repository contains all Microsoft Azure plugins for Gradle.
 
-Azure Plugin | Maven Central Version | Build Status
+Azure Plugin | Gradle Central Version | Build Status
```

Comment:
Gradle Central Version -> Gradle Plugin Portal
Consistency is important.

Reply:
Fixed